packages/fxa-content-server/app/scripts/templates/pair/auth_wait_for_supp.mustache

```diff
@@ -1,4 +1,4 @@
-<div id="main-content" class="card pair-auth">
+<div class="card">
```

Comment:
We appear to use this ID on a lot of other pages but looks like it styles a lot of inputs etc. so LGTM!